### PR TITLE
CSS elements guide

### PR DESCRIPTION
**Pull request by jesusbotella**
#238

### CODE REVIEW

File: docs/guides/01-css-elements.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# CSS Elements
+
```
Comment (IagoLast) [inline docs/guides/01-css-elements.md]: A small intro giving context will be nice

File: docs/guides/01-css-elements.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  @import '~@carto/airship-style';
+
+  // Import SASS element styles
+  @import '~@carto/airship-style/src/{path}';
+
+  // Import CSS element styles
+  @import '~@carto/airship-style/dist/{path}';
+  ```
+
+  #### Importing styles in JavaScript
```
Comment (IagoLast) [inline docs/guides/01-css-elements.md]: should we mention that a specific loader is required?

File: docs/guides/01-css-elements.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+  I'm a regular button
+</button>
+```
+
+Please check [styles documentation]() (missing link) out and see how to use Airship components.
+
+## Framework integration
+To integrate Airship into your framework you only need to import the styles in the way you prefer from the ones stated above.
+
+## Browser Compatibility
```
Comment (IagoLast) [inline docs/guides/01-css-elements.md]: Usually this section is at the end of the doc

File: docs/guides/01-css-elements.md (hunk 2 above)
Comment (jesusbotella) [reply docs/guides/01-css-elements.md]: I don't know, I'd say that we can assume the reader knows how to do that if they use Webpack.

File: docs/guides/01-css-elements.md (hunk 3 above)
Comment (jesusbotella) [reply docs/guides/01-css-elements.md]: We can put it at the end of the doc!

File: docs/guides/01-css-elements.md (hunk 2 above)
Comment (jesusbotella) [reply docs/guides/01-css-elements.md]: I added a warning just in case any users feel lost about it.

**Comment by IagoLast** [comment]
LGTM

**Pull request merged**